daemon/src/ipc_server.rs

```diff
@@ -146,6 +146,24 @@ pub fn handle_message(
             }
             IpcResponse::Ok
         }),
+
+        IpcMessage::TogglePauseWallpaper { monitors } => {
+            check_monitors(wpaperd, &monitors).map(|_| {
+                let surfaces = collect_surfaces(wpaperd, monitors);
+                if surfaces.is_empty() {
```

Comment:
In this case the `toggle` command does nothing if no monitor is passed. `pause` and `resume` instead apply for all monitors if none is passed. I am thinking that maybe having the same behaviour would be a little bit more intuitive to users.